website/content/docs/agent/telemetry.mdx

```diff
@@ -544,6 +544,7 @@ These metrics are used to monitor the health of the Consul servers.
 | `consul.xds.server.streamDrained`                   | Counts the number of xDS streams that are drained when rebalancing the load between servers.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | streams                           | counter |
 | `consul.xds.server.streamStart`                     | Measures the time taken to first generate xDS resources after an xDS stream is opened.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | ms                                | timer   |
+| `consul.xds.server.unauthenticated`                 | Counts the number of xDS streams that are unauthenticated because ACLs are not enabled or ACL tokens were missing.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | ms                                | timer   |
```

Comment:
`ms/timer` is not correct as implemented (with the counter) and would be more correct as `streams?/counter`

if you address the gauge suggestion then this should be `streams/gauge`

Reply:
Woops good catch! Copy paste error. Will probably set it to `streams/gauge` with your prior suggestion.